xbmc/video/test/TestVideoFileItemClassify.cpp

```diff
@@ -143,3 +143,10 @@ TEST(TestVideoFileItemClassify, IsSubtitle)
 
   EXPECT_FALSE(VIDEO::IsSubtitle(CFileItem("random.notasub", false)));
 }
+
+TEST(TestVideoFileItemClassify, IsVideoExtras)
+{
+  EXPECT_TRUE(VIDEO::IsVideoExtras(CFileItem("/home/foo/Extras/", true)));
+  EXPECT_TRUE(VIDEO::IsVideoExtras(CFileItem("/home/foo/extras/", true)));
+  EXPECT_FALSE(VIDEO::IsVideoExtras(CFileItem("/home/foo/Extraordinary/", true)));
```

Comment:
I'd suggest another test for "/home/foo/Extras/abc.mkv". Expected result false.

Reply:
added.